### PR TITLE
translate(pt/developer-docs/real-examples)

### PR DESCRIPTION
**Pull request by G3mha**
Finish translating the file `docs/pt/developer-docs/real-examples.md` to pt-BR.

Translates the files `docs/pt/developer-docs/real-examples/*` to pt-BR

### CODE REVIEW

File: docs/pt/developer-docs/real-examples/using-add-source-and-deltas-for-collectibles-and-similar-achievements.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Se você encontrar uma situação onde mais de uma bitflag em uma cadeia `AddSource` alterna ao mesmo tempo, você ainda pode utilizar este modelo. No entanto, você vai querer clonar as condições Delta e fazer com que ambos os totais criem um intervalo. Ainda usando a conquista acima como exemplo, o total existente poderia ser alterado para `<= 0x09` e o total para o segundo grupo poderia ser `>= 0x05`. Isso faria com que a conquista fosse ativada se o total passasse de 5-9 tesouros para 10 tesouros.
+
+---
+
+A Fazer:
+
+- Fornecer um exemplo separado para o intervalo Delta.
+- Abordar o uso de BitCount em uma conquista similar.
+- Abordar o uso de `/` quando você precisa de pelo menos uma de múltiplas coisas.
```
Comment (G3mha) [inline docs/pt/developer-docs/real-examples/using-add-source-and-deltas-for-collectibles-and-similar-achievements.md]: @wescopeland Can I turn these To-Dos in issues in the repository, and them remove this part on the original file and translations?

**Comment by wescopeland** [comment]
There's a similar image issue in these files as was discovered in https://github.com/RetroAchievements/docs/pull/152.

Definitely not your doing - it's from our ancient legacy docs. But while we're touching these files, it'd be very beneficial if we could move any external images into the repo itself.

**Review by G3mha — COMMENTED** [review]
Turned into issues!

**Comment by G3mha** [comment]
@wescopeland Ready for re-review. Turned the image link into a local image file.

**Comment by G3mha** [comment]
@gboquizosanchez I've updated several achievement editor images and their descriptions to follow the repo's best practices now, as instructed by @wescopeland. Since some of these pages have Spanish translations, would you mind reviewing the Spanish alt text changes to ensure they follow the translation standards for ES? Thanks!

**Review by wescopeland — APPROVED** [review]
cc @gboquizosanchez I will hold off on merging for 48 hours.

**Pull request merged**